libraries/AC_AttitudeControl/AC_PosControl.cpp

```diff
@@ -1421,6 +1414,24 @@ float AC_PosControl::crosstrack_error() const
     }
 }
 
+#if APM_BUILD_TYPE(APM_BUILD_ArduPlane)
```

Comment:
You shouldn't need this, if its not called copter will not include the code. Because the function is const calling it cannot mess-up anything in the pos controller.

Reply:
Sure, but Leonard *really* doesn't want this code on Copter and this ensures that :-)